accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -691,11 +692,11 @@ public void setRemoteVideoRenderer(BaseVideoRenderer renderer, boolean remoteScr
   /**
    * (Tries) to set the FPS of the shared video stream to the passed one. The FPS is rounded to
    * the nearest supported one.
-   * @param FPS
+   * @param framePerSecond
    */
-  public void setPublishingFPS(int FPS) {
+  public void setPublishingFPS(int framePerSecond) {
     LOG.d(LOG_TAG, "setSharingFPS: ", mPublisher);
-    Publisher.CameraCaptureFrameRate frameRate = getFPS(FPS);
+    Publisher.CameraCaptureFrameRate frameRate = getFPS(framePerSecond);
```

Comment:
1. We should also rename `getFPS` to `getFramesPerSecond` (and in general change `FPS` to `framesPerSecond` nomenclature across all accelerator files to be consistent)

2. Now I wonder is there any real advantage of introducing `framesPerSecond` name instead of following OpenTon SDK nomenclature `frameRate`? (this adds a bit of complexity when reading the code, but does it add any value?)